tests/detect-dce-opnum/test.rules

```diff
@@ -0,0 +1 @@
+alert tcp any any -> any any (msg:"DCERPC"; dce_opnum:4; sid:1;)
```

Comment:
The new notation for the keyword is `dce.opnum`. Sorry for not catching this earlier. Please use this. Most keywords with `_` in the name have been changed to this pattern. Latest docs will show you the correct latest usage.